docs/dev/api_usage_examples/common.py

```diff
@@ -0,0 +1,54 @@
+'''
+common.py
+'''
+
+from enum import Enum
+from time import time
+from datetime import datetime
+
+ROOT = '127.0.0.1'
+API_VERSION = 1
+TOKEN = '93fb3a42-9177-48a0-b07a-e8a7019218f6' # Replace with your dev token.
+USERNAME = 'naiveaiguy' # Replace with the username from which you got the token above.
+AUTH_HEADER = {
+    "Authorization": f"Token {TOKEN}"
```

Comment:
Use `.format` here. Python 3.6 fstrings cause problems when readthedocs builds the documentations.

Reply:
Sure.